docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,125 @@
 # Managing security and risk
```

Comment:
regarding use cases, I'd highlight:

1. Understand if the current state of security is good or bad.
2. Understand how security posture progresses over time.
3. Find and prioritize the most problematic areas or vulnerabilities that need attention.
4. Report status and progress to your stakeholders.

this presentation has more details about which section meets each use case: https://docs.google.com/presentation/d/1qyPTbfVPZJs7nqKRPm2H79Gl54bOpcHSAsJEJ5x1fu0/edit#slide=id.g2c087d3ee8e_0_42

Reply:
Thanks! Tweaked with more info on use cases ([commit](https://github.com/codacy/docs/pull/2052/commits/f3ead77a43017177c1f8ca8fdf835289aa782c29), [commit](https://github.com/codacy/docs/pull/2052/commits/3f60b6e6479645ef00dee3f5976327d822694f2b))